test/plugins/queue/scheduler.test.js

```diff
@@ -155,17 +155,13 @@ describe('scheduler test', () => {
 
     describe('startPeriodic', () => {
         beforeEach(() => {});
-        it("rejects if it can't establish a connection", function() {
+        it("does not reject if it can't establish a connection", async () => {
```

Comment:
any implication if connection error doesn't reject ? May be we should still reject on connection error after logging ?